### ISSUE TITLE
Community Spaces momentum a clear view on metrics to define success and ways to effectively interact with developers

### ISSUE THREAD

**Issue by brianking** [mozilla/participation-org#415]
## Objective Goal:

Our 3 community spaces are in full swing running events focused on developers, with a clear view on metrics to define success and ways to effectively interact with developers

R: Space Stewards
A: @brianking
C: @jmswisher 
I: @geroter 
## Tasks & To Dos Related to This Project:
- https://github.com/mozilla/community-spaces
## Sub-Goal Success:
- [ ] 3 month schedule for each space; Gather metrics from April; Mentor call for 2 more events types (R: @brianking @jmswisher Space Stewards A: @brianking C: I: )
- [ ] Build out further the more structured ways we want to interact with developers. This includes: sentiment post-event surveys ; folllow up email templates (R: Space Stewards @brianking A: @brianking C: I: ) 
## Related Issues & Links:
- https://wiki.mozilla.org/Marketing/Developer/Meetups_HOWTO

**Comment by brianking** [comment]
3 month schedule for each space; 
WIP here: https://docs.google.com/a/mozilla.com/spreadsheets/d/1h8nsKT00FVrRLUu1-2hpM-p0GD3eHztYB5AqdKm0suw/edit
Each space has their schedule, and we pulling it together into one place. While we figure out the right tool (talking with Pierros and team), we'll use a spreadsheet for now.

Gather metrics from April; 
Monthly reports here: https://drive.google.com/drive/folders/0Bw8tHLgpWV_POUdBMmpUZUhMc1E

Mentor call for 2 more events types;
Calls recorded here: https://air.mozilla.org/?tag=spaces

**Comment by brianking** [comment]
Registration and surveys are being covered here:
https://github.com/mozilla/community-spaces/issues/3
tl;dr Each space has their own system currently, but we are looking for the right tool to standardise one one for all.

**Comment by brianking** [comment]
We will continue in #424
